packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,229 +1,412 @@
-  cursor: pointer;
-}
-
-.aa-LoadingIndicator {
-  position: absolute;
-  z-index: 2;
-}
-
-.aa-LoadingIcon {
-  height: 100%;
-}
-
-.aa-List {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-}
-
-.aa-Item {
-  color: #23263b;
-  cursor: pointer;
-  display: flex;
-  grid-gap: 0.5rem;
-  justify-content: space-between;
-
-  mark {
-    background: none;
-    font-style: normal;
-    font-weight: bold;
-  }
-
-  /* stylelint-disable-next-line */
-  &[aria-selected='true'] {
-    background-color: #f5f5fa;
-  }
-}
-
-.aa-ItemLink {
-  color: inherit;
-  text-decoration: none;
-}
-
-.aa-ItemContent {
-  display: flex;
-  flex-grow: 1;
-  grid-gap: 0.5rem;
-  padding: 0.5rem;
-}
-
-.aa-ItemSourceIcon,
-.aa-ItemActionButton {
-  color: rgba(0, 0, 0, 0.4);
-}
-
-.aa-ItemActionButton {
-  background-color: transparent;
-  border: 0;
-  cursor: pointer;
-  opacity: 0.8;
-  padding: 0.5rem;
-  transition: color 100ms;
-  @media screen and (prefers-reduced-motion: reduce) {
-    transition: none;
-  }
-
-  &:hover,
-  &:focus {
-    color: rgba(80, 80, 80, 1);
-  }
+// ----------------
+// Darkmode
+// ----------------
+body {
+    &[data-theme='dark'],
+    &.dark {
+        --primary-color: rgb(247, 247, 0);
+        --muted-color: rgba(167, 167, 96, 0.6);
+        --selected-color: rgba(247, 247, 0, 0.1);
+        --icon-color: rgb(31, 207, 248);
+        --text-color: rgb(183, 192, 199);
+        --background-color: rgb(21, 23, 42);
+        --panel-shadow: inset 1px 1px 0 0 rgb(44, 46, 64), 0 3px 8px 0 rgb(0, 3, 9);
+    }
 }
 
-.aa-TouchOverlay {
-  backdrop-filter: blur(16px);
-  background: rgba(255, 255, 255, 0.95);
-  bottom: 0;
-  left: 0;
-  padding: 0.5rem;
-  position: fixed;
-  right: 0;
-  top: 0;
+// ----------------
+// Autocomplete
+// ----------------
+.aa-Autocomplete {
+    font-size: var(--font-size);
+    text-align: left;
+    line-height: 1em;
+    //reset
+    * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+    }
+    button {
+        appearance: none;
+        background: none;
+        border: none;
+    }
+    // searchbox
+    form.aa-Form {
+        position: relative;
+        display: flex;
+        border: 1px solid var(--muted-color);
+        align-items: center;
+        padding: 0 var(--spacing) 0 var(--spacing-half);
+        line-height: 1em;
+        background-color: var(--background-color);
+        border-radius: 3px;
+        &,
+        &.aa-Form--focused {
+            border-color: var(--primary-color);
+            box-shadow: var(--selected-color) 0 0 0 3px, inset var(--selected-color) 0 0 0 2px;
+            outline: currentcolor none medium;
+        }
+        .aa-InputWrapperPrefix {
+            order: 1;
+            flex-shrink: 0;
+            flex-wrap: none;
+            display: flex;
+            align-items: center;
+            padding-right: var(--spacing-half);
+            // the container for search and loading icons
+            label.aa-Label,
```

Comment:
```suggestion
            .aa-Label,
```

Reply:
Yes, i voluntary kept the tag name to help read the style. It's not optimized for the output. I believe it might be useful but it certainly doesn't look good from optimization side. Do you think it's a bad point?

Reply:
More selectors should mean more work from the browser so it would normally be faster and less redundant to only have `.aa-Label`

Reply:
yes it doesn't really have an impact there, the size and the perf could be optimized for sure but it does not really change the front-end experience. Question is does it impact the DX in a right way like 'oh, everything is very well structured and styling is obvious' or more 'wow, they really don't know how to write a good stylesheet'.

Reply:
The DX benefit that I see on our side is that we see that it's applied to a label in the editor:

![image](https://user-images.githubusercontent.com/6137112/106002213-27e92200-60b1-11eb-93fa-c096b84f11de.png)

But usually we aim for the lower specificity and providing `label` increases it. It adds weight and becomes harder to override.

I think removing them provides a better user-land DX actually?